puzzle.js

```diff
@@ -313,7 +319,18 @@ function PuzzleEntry(p) {
     }
 
     this.onUndoRedo = function(units) {
-        units.forEach((u) => { if (u.attribute == "data-path-code" || u.attribute == "data-edge-code" || u.attribute == "data-x-edge-code") { this.updateSvg(u.elem); }});
+        units.forEach((u) => {
+            if (u.attribute == "data-path-code" || u.attribute == "data-edge-code" || u.attribute == "data-x-edge-code") {
+                this.updateSvg(u.elem);
+            }
+            if (u.elem instanceof HTMLTableCellElement) {
+                this.processTdForCopyjack(u.elem);
+            } else if (u.elem.parentElement instanceof HTMLTableCellElement) {
+                this.processTdForCopyjack(u.elem.parentElement); // TODO(jhimawan): fix after no-input refactor.
```

Comment:
This is probably correct actually, since the text needs to be on top of the SVG.

Reply:
Is action needed? Sorry, I don't think I understand what you mean.

Reply:
I just meant you could remove the comment since the post-<input> version will still want to do a parent check.